### PR TITLE
🐛 Improve a bug issue template

### LINKED ISSUE #230: Github templates are not relevant to the project
See the template below.
Multidict has no button to click and has no OS/Browser specific

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by serhii73**
Resolves #230

### CODE REVIEW

**Comment by webknjaz** [comment]
Please make commit message/pr title follow best practices of writing commits, like:
```
🐛 Improve a bug issue template

Resolves #230
```

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 1]
```diff
@@ -8,10 +8,12 @@ about: Create a report to help us improve
 A clear and concise description of what the bug is.
 
 **To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
+A clear and concise description of steps to reproduce the behavior.
```
Comment (webknjaz) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: I suggest putting this inside of html comments, i.e. `<!-- whole examples/explanation text here -->`

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 2]
```diff
@@ -8,10 +8,12 @@ about: Create a report to help us improve
...
 
 **To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
+A clear and concise description of steps to reproduce the behavior.
+
+For example:
+1. Open Terminal
```
Comment (webknjaz) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: Better:
1. Have certain environment
1. Run given code snippet in a certain way
1. See some behavior described

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 3]
```diff
@@ -8,28 +8,18 @@ about: Create a report to help us improve
 A clear and concise description of what the bug is.
 
 **To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+<!-- whole examples/explanation text here -->
```
Comment (webknjaz) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: What I meant was that we should *hide* the example part and `Steps to reproduce the behavior:` with a comment, not write that dummy string above.

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 4]
```diff
@@ -8,28 +8,18 @@ about: Create a report to help us improve
...
-4. See error
+<!-- whole examples/explanation text here -->
+
+For example:
+1. Have certain environment
+2. Run given code snippet in a certain way
+3. See some behavior described
 
 **Expected behavior**
 A clear and concise description of what you expected to happen.
```
Comment (webknjaz) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: Apply the same commenting technique here and in other places.

**Comment by webknjaz** [comment]
@asvetlov Andrew do you have any additional comments on this?

**Comment by asvetlov** [comment]
I very doubt if screenshot section is needed.
But a section with explicit ask for used multidict and python versions would be helpful.

**Comment by webknjaz** [comment]
@serhii77 so replace screenshots with logs/tracebacks and add questions about multidict distribution and python versions.

**Comment by serhii73** [comment]
@webknjaz 

> also have explicit python and explicit multidict questions

Sorry?

**Comment by webknjaz** [comment]
@serhii73 I mean, ask separate questions for each of them

**Comment by webknjaz** [comment]
@serhii73 also apply the same title/description style for the PR itself as for the commits (https://github.com/aio-libs/multidict/pull/246#issuecomment-392324709)

**Comment by webknjaz** [comment]
@serhii77 will done!

**Pull request merged**